package.json

```diff
@@ -7,10 +7,17 @@
     "build": "run-s build:css build:res",
     "dev:res": "bsb -make-world -w",
     "dev:css": "./node_modules/.bin/postcss ./src/index.css -o ./theme/assets/index-dev.css -w",
-    "dev": "run-p dev:css dev:res",
-    "clean": "bsb -clean-world",
-    "pull": "cd theme && theme download",
-    "watch": "cd theme && theme watch --allow-live"
+    "dev": "run-p dev:css dev:res dev:watch",
```

Comment:
@nireno `./local.sh watch development` will already be running `dev:css` & `dev:res`. Let's remove this `dev` script and rename `dev:watch` to `dev` since that's what we use in other projects for local development